include/builtin.h

```diff
@@ -262,18 +258,11 @@ template <typename Impl, typename ClassPolicy = DefaultClassPolicy> class Builti
   }
 
   static void register_subclass(const JSClass *cls) {
-    get_registry().push_back(cls);
+    get_registry().insert(cls);
   }
 
   static bool is_subclass(const JSObject *obj) {
-    if (obj == nullptr)
-      return false;
-
-    const JSClass *cls = JS::GetClass(obj);
-    const auto &registry = get_registry();
-
-    return std::any_of(registry.begin(), registry.end(),
-                       [cls](const auto *reg) { return reg == cls; });
+    return std::ranges::any_of(get_registry(), [&](auto *cls) { return cls == JS::GetClass(obj); });
```

Comment:
Is there any reason not to use `get_registry().contains()`? As far as I can tell, `any_of` still has to do a linear search, since AFAICT the condition has to be tested for each item.